ext/bg/js/dictionary-importer.js

```diff
@@ -199,6 +193,25 @@ class DictionaryImporter {
         return {result: summary, errors};
     }
 
+    _createSummary(dictionaryTitle, version, index, details) {
+        const summary = {
+            title: dictionaryTitle,
+            revision: index.revision,
+            sequenced: index.sequenced,
+            version
+        };
+
+        const {author, url, description, attribution} = index;
+        if (typeof author === 'string') { summary.author = author; }
+        if (typeof url === 'string') { summary.url = url; }
+        if (typeof description === 'string') { summary.description = description; }
+        if (typeof attribution === 'string') { summary.attribution = attribution; }
```

Comment:
The type check here might be redundant if it's possible to assign the entire `index` into `summary` as-is because it's already checked in the schema

Reply:
Yeah, and that would also allow unexpected data to be stored in the database, since the schema permits extra properties.